meetings/2024-10-09.md

```diff
@@ -30,15 +39,111 @@
+* ...: a new protocol (binary) not using HTTP, no domain names and certs, also for efficiency
+* ...: we are collaborating with ActivityPods, already mentioned today, yesterday we annonced the collabortion. On Monday activitypods released v2.0 - it combines ActivityPub with Solid.
+* ...: ActivityPods also uses SPARQL, solid documents are named graphs. Next Graph is doing the same which fits well. They will also adopt DIDs for documents and agents, as NextGraph. A special DID method will be registered.
+* ...: Activity Pods are using SAI - Solid Application Interop, NextGraph is also using it.
+* ...: long term goal is unify our frameworks
+* ...: reactive stores in React, Svelte, Vue
+* ...: NextGraph has documents which are RDF resources, which have primary class.
+* ...: viewers / editors are the apps, manifest specifies which primary classes it can read and write
+* ...: end user can choose which viewer / editor it uses, user is the owenr of the data
+* ...: plans for everything app, to give one good app, there can be other apps available but we want to have one good app on every domain.
+* ...: it is also a social network thanks to pub/sub, it can be compatible with activitypub, also can be bridged to other protocols
+* ...: data is based on revisions, each change is a commit, one can come back to any past revision
+* ...: content is signed by the author, also there are group signatures to proof authenticity to the outsiders
+* ...: it also has branches, new branches can be created as blocks, document can be composed of blocks
+* ...: we also have consensus based on threshold signatures, similar to how blockchains work
+* ...: synchronous transactions (ACID)
+* ...: thanks to NLnet for supporting the project
+* ...: engaging in talks with other actors to converge and gather forces to defeat Big Tech
+* ...: demo!
+* ...: we have web version on left and native linux on the right. there are three stores, one is private which user can only share across devices. there is also the shared space, where one can interact with other users. There is also a public space, like a website.
+* ...: I will create a new document, we can select from different primary classes. As you see it synchronized. I can change the title, it also updated in the other app. I can do a SPARQL update.
+* ...: you can see `did:ng:o:.....` the last part has a public key. 
+* ...: let's do another update, now in the other app we see the whole history of the document, creation, signature, adding first triple, then adding another triple. I will add another one on the left and we can see another commit on the right. 
+* ...: Now I will disable connection in the web app, I can still edit the document. And in the other ap I will make a different update. No sync yet since the webapp is offline. Now we go backonline and we see on the right a fork. As soon as I make another commit we can see the automatic merge. So it has the automerge without conflicts, it also work on data deletion.
```

Comment:
```suggestion
* ...: Let's do another update. Now, in the other app, we see the whole history of the document: creation, signature, adding first triple, then adding another triple. I will add another one on the left, and we can see another commit on the right. 
* ...: Now I will disable connection in the web app, I can still edit the document. And in the other app, I will make a different update. No sync yet since the web app is offline. Now we go back online, and we see on the right a fork. As soon as I make another commit, we can see the automatic merge. So it has the automerge without conflicts. It also works on data deletion.
```